.taskfiles/tools/gofumpt.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+
+tasks:
+  install:
+    desc: A stricter formatter than gofmt
+    dir: '{{.ROOT_DIR}}'
+    internal: true
+    requires:
+      vars:
+        - GOFUMPT_VERSION
+    status:
+      - test -f $GOBIN/gofumpt
+    cmds:
+      - go install mvdan.cc/gofumpt@{{.GOFUMPT_VERSION}}
+
+  gci:
```

Comment:
Nit: This task name should probably be `gofumpt`?

Reply:
Ah copy & paste error!